File: javascript-axios/examples/get_filing_products_offerings.js
const path = require('path');
process.chdir(path.resolve(__dirname, '../../'));
require('dotenv').config();

const COMPANY_ID = process.env.COMPANY_ID;
const JURISDICTION = process.env.JURISDICTION;

params = {
	company_id: COMPANY_ID,
	jurisdiction: JURISDICTION
}

token = require('../request.js').request.token({ path: '/filing-products/offerings' });

require('../request.js').request.get({ path: '/filing-products/offerings', token: token, queryParams: params });

File: csharp/examples/GetFilingProductsOfferings.cs
using System;

namespace Examples.examples
{
    // Example of GET /filing-products/offerings
    
    class GetFilingProductsOfferings : BaseRequest
    {
		private string _companyId;
        private string _jurisdiction;

        public GetFilingProductsOfferings()
		{
            dotenv.net.DotEnv.Load();
            _companyId      = Environment.GetEnvironmentVariable("COMPANY_ID");
            _jurisdiction   = Environment.GetEnvironmentVariable("JURISDICTION");
            Console.WriteLine($"GetFilingProductsOfferings: _companyId={_companyId} _jurisdiction={_jurisdiction}");
        }

        public GetFilingProductsOfferings(string companyId, string jursidiction)
        {
            _companyId = companyId;
            _jurisdiction = jursidiction;
        }

        public override string SendRequest()
        {
            string path = "filing-products/offerings";
            string queryParams = $"?company_id={_companyId}&jurisdiction={_jurisdiction}";
            return GetRequest(path + queryParams);
        }
	}
}